docs/project.md

```diff
@@ -173,8 +173,8 @@ The default is set according to the following table:
 
 | `reuse_private_key` value | Default `keyname`         |
 | ------------------------- | ------------------------- |
-| `false`                   | `{{prefix}}_{{version}}`  |
-| `true`                    | `{{prefix}}`              |
+| `false`                   | `{{prefix}}}_{{version}}`  |
```

Comment:
Triple `}}}`? :thinking:

Reply:
Find all and replace mistake, fixed in next commit